pkg/addons/validations.go

```diff
@@ -93,3 +95,15 @@ func contains(slice []string, val string) bool {
 	}
 	return false
 }
+
+// SupportsAmd64 ensures that the cluster supports running amd64 images
+func SupportsAmd64(cc *config.ClusterConfig, name, _ string) error {
```

Comment:
lets see if we can run it through translations

Reply:
lets investigate what are the steps to do to run through translation

Reply:
arm64 can be supported by installing nerdctl and running:
```
sudo nerdctl run --privileged --rm tonistiigi/binfmt --install all
```